### ISSUE TITLE
crashes when viewing panorama (photosphere)

### ISSUE THREAD

**Issue by sunhasgothishaton** [SimpleMobileTools/Simple-Gallery#2255]
Hi

I click to view a panorama (photosphere) that is 360 degrees x 180 degrees.

An icon is displayed (an arrow that appears to go 360 degrees). When I  click on it, an error is displapyed "Unfortunately, gallery has stopped".

Is there a way I can get it working?

Thanks

**Comment by tibbi** [comment]
what app and OS version are you using? Do other apps display it properly?

**Comment by sunhasgothishaton** [comment]
I have decided to go with another application.

Thank you.

**Comment by sunhasgothishaton** [comment]
so

**Comment by tibbi** [comment]
okay, will keep an eye on it then

**Comment by sunhasgothishaton** [comment]
After further testing of 4 360 panorama viewers :

1 worked for me, not crashing and displaying the panoramas correctly.

The others
- two of them crashed when trying to display them
- one displayed one correctly, skewed one, crashed on others

So maybe its my tablet, rather than the gallery app?

**Comment by tibbi** [comment]
Im not sure, could be some low memory too. Try restarting your device.

**Comment by sunhasgothishaton** [comment]
I rebooted the device, no difference.

I then scaled the images down (8192 pixels wide and 4096 pixels wide, both 80 quality level when saving jpeg in gimp). One app could display the original images, one app could use the 8192px wide images, another could use the 4096px wide images. gallery couldn't display any of these, other 360 panorama apps couldn't display them either, they crashed in the process of displaying them.

I installed updates from google play store, none of the apps  were updated, thing like google play store and google services were. now only one app could display the original images, one app could display 8192px wide images, the one that could display 4096px wide images couldn't any more.

I hard reset my tablet, installed play store updates. installed apps. I have two apps that display the 360 panoramas, one that displays the originals and one that displays 8196px wide images.

I tried on a family members tablet (android 4.4.2), however some apps weren't available for this android version (including gallery). No apps crashed, just failed to load with the loading disc spinning. More apps loaded 360 panoramas on this tablet than on mine.

As several apps are crashing on my tablet, I think that I will keep the app that is working for my use.

I am finding lately that more and more apps aren't working on my tablet, not just this one feature of gallery. many that used to work now crash when starting. So an update (somewhere) seems has caused this.

Thanks for your help,
Rob
